services/management/dbaas/version_service_client.go

```diff
@@ -146,3 +154,52 @@ func (c *VersionServiceClient) Matrix(ctx context.Context, params componentsPara
 
 	return &vsResponse, nil
 }
+
+func getLatest(m map[string]componentVersion) (*goversion.Version, error) {
+	if len(m) == 0 {
+		return nil, errNoVersionsFound
+	}
+	keys := make([]*goversion.Version, len(m))
+	i := 0
+	var err error
+	for k := range m {
+		keys[i], err = goversion.NewVersion(k)
+		if err != nil {
+			return nil, err
+		}
+		i++
+	}
+
+	latest := goversion.Must(goversion.NewVersion("v0.0.0"))
+	for _, version := range keys {
+		if version.GreaterThan(latest) {
+			latest = version
+		}
+	}
```

Comment:
it can be done using only one loop instead of two.

Reply:
done